account/manager.go

```diff
@@ -1754,105 +1818,112 @@ func valueAfterAccountUpdate(account *Account, outputs []*wire.TxOut,
 // the inputs is also provided to use when coming across an unexpected failure.
 // If needed, a change output from the backing lnd node's wallet may be returned
 // as well.
-func (m *manager) inputsForDeposit(ctx context.Context,
-	depositAmount btcutil.Amount, witnessType witnessType,
-	feeRate chainfee.SatPerKWeight) ([]chanfunding.Coin, func(),
-	*wire.TxOut, error) {
+func (m *manager) inputsForDeposit(ctx context.Context, account *Account,
+	newAccountOutput *wire.TxOut, depositAmount btcutil.Amount,
+	witnessType witnessType, feeRate chainfee.SatPerKWeight) (*psbt.Packet,
+	func(), error) {
+
+	// Unfortunately the FundPsbt call doesn't allow us to specify _any_
+	// inputs, otherwise it won't perform coin selection at all. So what we
+	// do instead is to fund our account output just for the funding amount
+	// plus whatever we need to pay for the additional input (which we know
```

Comment:
Nice work around! We'll probably eventually patch this behavior, but good to be abble to get by w/ less as well.